spharpy/spherical.py

```diff
@@ -248,6 +248,104 @@ def n3d_to_sn3d_norm(n):
     return 1 / np.sqrt(2 * n + 1)
 
 
+def renormalize(data, channel_convention, current_norm, target_norm, axis=-2):
```

Comment:
```suggestion
def renormalize(data, channel_convention, current_norm, target_norm, axis):
```
Using -2 as default axis does not seem like the best option here. For example for a basis matrix it is -1.
So I would prefer forcing users to choose the appropriate axis.